recipes/colt5-attention/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "colt5-attention" %}
+{% set version = "0.10.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/CoLT5-attention-{{ version }}.tar.gz
+  sha256: a99c2b5de68bdb5b814329ea7aba2d08778942f18d2d2d1382721400394d3656
+
+build:
+  skip: true  # [win]
```

Comment:
This one can probably be noarch, even with the missing pytorch on Windows. The solver will prevent its installation there and, then pytorch is available on Windows, this package will be ready without the need for any modification.